[flaky] TestCommitterReleaseImagesWithTLS

```
=== Failed
=== FAIL: docker/test TestCommitterReleaseImagesWithTLS/database:yugabyte/tls-mode:mtls (15.09s)
    container_release_image_test.go:93: network sc_test_network_1fee10bf-3441-4fc5-a927-01cb2152b0b0 created
2026-02-23 16:33:21.428 UTC [grpc-connection] ] -> INFO Closing 1 connections.
    container_release_image_test.go:109: container 'sc_test_verifier_mtls_yugabyte' not found
2026-02-23 16:33:21.430 UTC [grpc-connection] ] -> INFO Closing 1 connections.
    container_release_image_test.go:109: container 'sc_test_vc_mtls_yugabyte' not found
2026-02-23 16:33:21.431 UTC [grpc-connection] ] -> INFO Closing 1 connections.
    container_release_image_test.go:109: container 'sc_test_query_mtls_yugabyte' not found
2026-02-23 16:33:21.434 UTC [grpc-connection] ] -> INFO Closing 1 connections.
    container_release_image_test.go:109: container 'sc_test_coordinator_mtls_yugabyte' not found
2026-02-23 16:33:21.436 UTC [grpc-connection] ] -> INFO Closing 1 connections.
    container_release_image_test.go:109: container 'sc_test_sidecar_mtls_yugabyte' not found
2026-02-23 16:33:21.438 UTC [grpc-connection] ] -> INFO Closing 1 connections.
    container_release_image_test.go:109: container 'sc_test_db_mtls_yugabyte' not found
2026-02-23 16:33:21.440 UTC [grpc-connection] ] -> INFO Closing 1 connections.
    container_release_image_test.go:109: container 'sc_test_orderer_mtls_yugabyte' not found
2026-02-23 16:33:21.442 UTC [grpc-connection] ] -> INFO Closing 1 connections.
    container_release_image_test.go:109: container 'sc_test_loadgen_mtls_yugabyte' not found
    container_release_image_test.go:115: 
        	Error Trace:	/home/runner/work/fabric-x-committer/fabric-x-committer/service/vc/dbtest/container.go:109
        	            				/home/runner/work/fabric-x-committer/fabric-x-committer/docker/test/container_release_image_test.go:152
        	            				/home/runner/work/fabric-x-committer/fabric-x-committer/docker/test/container_release_image_test.go:115
        	Error:      	Received unexpected error:
        	            	starting container: API error (500): failed to set up container networking: driver failed programming external connectivity on endpoint sc_test_yugabyte_unit_tests_with_tls_fa7d585c (cb41eeda99402bb9237efacea6b4a92b96c6cfd910ac5be3ac054d0c4a027996): failed to bind host port for 0.0.0.0::172.18.0.2:5433/tcp: address already in use
        	            	(1) attached stack trace
        	            	  -- stack trace:
        	            	  | github.com/hyperledger/fabric-x-committer/service/vc/dbtest.(*DatabaseContainer).start
        	            	  | 	/home/runner/work/fabric-x-committer/fabric-x-committer/service/vc/dbtest/container.go:124
        	            	  | github.com/hyperledger/fabric-x-committer/service/vc/dbtest.(*DatabaseContainer).StartContainer
        	            	  | 	/home/runner/work/fabric-x-committer/fabric-x-committer/service/vc/dbtest/container.go:109
        	            	  | github.com/hyperledger/fabric-x-committer/docker/test.startSecuredDatabaseNode
        	            	  | 	/home/runner/work/fabric-x-committer/fabric-x-committer/docker/test/container_release_image_test.go:152
        	            	  | github.com/hyperledger/fabric-x-committer/docker/test.TestCommitterReleaseImagesWithTLS.func1.1
        	            	  | 	/home/runner/work/fabric-x-committer/fabric-x-committer/docker/test/container_release_image_test.go:115
        	            	  | testing.tRunner
        	            	  | 	/opt/hostedtoolcache/go/1.25.5/x64/src/testing/testing.go:1934
        	            	  | runtime.goexit
        	            	  | 	/opt/hostedtoolcache/go/1.25.5/x64/src/runtime/asm_amd64.s:1693
        	            	Wraps: (2) starting container
        	            	Wraps: (3) API error (500): failed to set up container networking: driver failed programming external connectivity on endpoint sc_test_yugabyte_unit_tests_with_tls_fa7d585c (cb41eeda99402bb9237efacea6b4a92b96c6cfd910ac5be3ac054d0c4a027996): failed to bind host port for 0.0.0.0::172.18.0.2:5433/tcp: address already in use
        	            	Error types: (1) *withstack.withStack (2) *errutil.withPrefix (3) *docker.Error
        	Test:       	TestCommitterReleaseImagesWithTLS/database:yugabyte/tls-mode:mtls
    container_release_image_test.go:95: network sc_test_network_1fee10bf-3441-4fc5-a927-01cb2152b0b0 removed successfully
        --- FAIL: TestCommitterReleaseImagesWithTLS/database:yugabyte/tls-mode:mtls (15.09s)

=== FAIL: docker/test TestCommitterReleaseImagesWithTLS/database:yugabyte (0.00s)
    --- FAIL: TestCommitterReleaseImagesWithTLS/database:yugabyte (0.00s)

=== FAIL: docker/test TestCommitterReleaseImagesWithTLS (0.07s)
    container_release_image_test.go:67: creating config-block

DONE 14 tests, 3 failures in 123.570s
make: *** [Makefile:160: test-container] Error 1
Error: Process completed with exit code 2.
```

--- a/service/vc/dbtest/container.go
+++ b/service/vc/dbtest/container.go
@@ -315,7 +315,7 @@ func (dc *DatabaseContainer) ExposePort(port string) {
 	if dc.PortBinds == nil {
 		dc.PortBinds = make(map[docker.Port][]docker.PortBinding)
 	}
-	dc.PortBinds[p] = []docker.PortBinding{{HostIP: "0.0.0.0", HostPort: ""}}
+	dc.PortBinds[p] = []docker.PortBinding{{HostIP: "0.0.0.0", HostPort: "0"}}
 }
 
 // GetHostMappedEndpoint inspects the container and returns a host-accessible endpoint

